### PR TITLE
Added radiation reading and core tray to sample detail

### PR DESCRIPTION
**Pull request by DavidArmstrongGSQ**

### CODE REVIEW

File: uat-vocabularies/sample-detail-type.ttl [hunk 1]
```diff
@@ -52,3 +52,17 @@ smpdt:core-tray-condition a skos:Concept ;
...
+skos:inScheme https://linked.data.gov.au/def/sample-detail-type ; 
+skos:prefLabel "Radiation reading"@en ; 
+skos:topConceptOf https://linked.data.gov.au/def/sample-detail-type .
+
+smpdt:core-tray a skos:Concept ; 
+skos:definition "The range of plastic, cardboard or metal tray which holds samples of drill core on a pallet”@en ;
+rdfs:isDefinedBy https://linked.data.gov.au/def/sample-detail-type ; 
+skos:inScheme https://linked.data.gov.au/def/sample-detail-type ; 
+skos:prefLabel "Core tray"@en ; 
+skos:topConceptOf https://linked.data.gov.au/def/sample-detail-type .
```
Comment (KellyVance) [inline uat-vocabularies/sample-detail-type.ttl:68]: change https://linked to http://linked for all occurrences in the uat-vocabulary sub-repo

```suggestion
smpdt:radiation-reading a skos:Concept ; 
skos:definition "Radiation reading collected of each pallet, units recorded in microsievert per hour (μSv/hour) "@en ;
rdfs:isDefinedBy http://linked.data.gov.au/def/sample-detail-type ; 
skos:inScheme http://linked.data.gov.au/def/sample-detail-type ; 
skos:prefLabel "Radiation reading"@en ; 
skos:topConceptOf http://linked.data.gov.au/def/sample-detail-type .

smpdt:core-tray a skos:Concept ; 
skos:definition "The range of plastic, cardboard or metal tray which holds samples of drill core on a pallet”@en ;
rdfs:isDefinedBy https://linked.data.gov.au/def/sample-detail-type ; 
skos:inScheme https://linked.data.gov.au/def/sample-detail-type ; 
skos:prefLabel "Core tray"@en ; 
skos:topConceptOf https://linked.data.gov.au/def/sample-detail-type .
```

**Review by KellyVance — CHANGES_REQUESTED** [review]
Change to http in uat required
Need change to mod date
also good practice to include top concepts in the concept scheme top concept list.

File: vocabularies/sample-detail-type.ttl [hunk 2]
```diff
@@ -52,3 +52,17 @@ smpdt:core-tray-condition a skos:Concept ;
...
+skos:inScheme https://linked.data.gov.au/def/sample-detail-type ; 
+skos:prefLabel "Radiation reading"@en ; 
+skos:topConceptOf https://linked.data.gov.au/def/sample-detail-type .
+
+smpdt:core-tray a skos:Concept ; 
+skos:definition "The range of plastic, cardboard or metal tray which holds samples of drill core on a pallet”@en ;
+rdfs:isDefinedBy https://linked.data.gov.au/def/sample-detail-type ; 
+skos:inScheme https://linked.data.gov.au/def/sample-detail-type ; 
+skos:prefLabel "Core tray"@en ; 
+skos:topConceptOf https://linked.data.gov.au/def/sample-detail-type .
```
Comment (KellyVance) [inline vocabularies/sample-detail-type.ttl:68]:
```suggestion
smpdt:radiation-reading a skos:Concept ; 
skos:definition "Radiation reading collected of each pallet, units recorded in microsievert per hour (μSv/hour)"@en ;
rdfs:isDefinedBy <https://linked.data.gov.au/def/sample-detail-type> ; 
skos:inScheme <https://linked.data.gov.au/def/sample-detail-type> ; 
skos:prefLabel "Radiation reading"@en ; 
skos:topConceptOf <https://linked.data.gov.au/def/sample-detail-type> .

smpdt:core-tray a skos:Concept ; 
skos:definition "The range of plastic, cardboard or metal tray which holds samples of drill core on a pallet"@en ;
rdfs:isDefinedBy <https://linked.data.gov.au/def/sample-detail-type> ; 
skos:inScheme <https://linked.data.gov.au/def/sample-detail-type> ; 
skos:prefLabel "Core tray"@en ; 
skos:topConceptOf <https://linked.data.gov.au/def/sample-detail-type> .

```

File: uat-vocabularies/sample-detail-type.ttl (hunk 1 above)
Comment (LukeHauck) [reply uat-vocabularies/sample-detail-type.ttl:68]: Files good with these amendments.

**Pull request merged**